include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -33,11 +49,17 @@ class BrushedMotorInterruptHandler {
     BrushedMotorInterruptHandler(
         GenericQueue& incoming_queue, StatusClient& outgoing_queue,
         motor_hardware::BrushedMotorHardwareIface& hardware_iface,
-        brushed_motor_driver::BrushedMotorDriverIface& driver_iface)
+        brushed_motor_driver::BrushedMotorDriverIface& driver_iface,
+        lms::LinearMotionSystemConfig<lms::GearBoxConfig>& gearbox_config)
         : queue(incoming_queue),
           status_queue_client(outgoing_queue),
           hardware(hardware_iface),
-          driver_hardware(driver_iface) {}
+          driver_hardware(driver_iface),
+          gear_conf(gearbox_config) {
+        acceptable_position_error =
+            int32_t(gear_conf.get_encoder_pulses_per_mm() *
```

Comment:
Should this also be absolute value since we're comparing it below to the abs of the `move_delta`?